travis/push-to-pypi.sh

```diff
@@ -30,12 +30,16 @@ WHEEL_NAME="acc_provision-${TRAVIS_TAG}.tar.gz"
 TAG_NAME="acc_provision-${TRAVIS_TAG}"
 DEV_WHEEL_NAME="acc_provision-${TRAVIS_TAG}.dev${TRAVIS_BUILD_NUMBER}.tar.gz"
 DEV_TAG_NAME="acc_provision-${TRAVIS_TAG}.dev${TRAVIS_BUILD_NUMBER}"
+TWINE_UPLOAD="true"
 
 if [ -n "$PYPI_RELEASE" ] ; then
     #twine upload --repository-url https://pypi.org/legacy/ -u ${PYPI_USER} -p ${PYPI_PASS} dist/$WHEEL_NAME
     python setup.py sdist
     twine upload -u ${PYPI_USER} -p ${PYPI_PASS} dist/$WHEEL_NAME
-    $SCRIPTS_DIR/push-to-cicd-status.sh "https://pypi.org/project/acc-provision/"${TRAVIS_TAG}"/#files" "${TAG_NAME}"
+    if [ $? -ne 0 ]; then
+        TWINE_UPLOAD="false"
```

Comment:
May be we can use same name TWINE_UPLOAD everywhere  - even in push-to-cicd-status.sh as TWINE_UPLOAD to have same meaning everywhere